pkg/instrumentation/helper.go

```diff
@@ -14,7 +14,30 @@
 
 package instrumentation
 
-import corev1 "k8s.io/api/core/v1"
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func defaultInitContainerResourceRequirements() corev1.ResourceRequirements {
+	const (
+		initContainerCPULimit = "1"
+		initContainerMEMLimit = "256Mi"
```

Comment:
How big are the agents? I am not sure if any is bigger than 100mb.

Reply:
Iam assuming there that the agent size is almost equal to the image size.
```
autoinstrumentation-java     078769bc3266   4 days ago     34.1MB
autoinstrumentation-nodejs   1bcc3acff819   11 days ago    114MB
autoinstrumentation-python   4e612ec06244   11 days ago    19MB
autoinstrumentation-dotnet   7a0228b7eafc   11 days ago    76.4MB
```